cd/helm/CAPIF/templates/api-invocation-logs-service.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    kompose.cmd: kompose convert -f docker-compose-resolved.yaml -c -o folder
+    kompose.version: 1.26.1 (HEAD)
+  creationTimestamp: null
+  labels:
+    io.kompose.service: api-invocation-logs
+  name: api-invocation-logs
+spec:
+  ports:
+    - name: "8080"
```

Comment:
Change Name self-generated 8080 makes some problems at route yaml of service because it will be understand like number not like name. Maybe we can set something like <service_name>-port